### DISCUSSION TITLE
Simply view the resource w/o edit options

### DISCUSSION THREAD

**Discussion by vv-ac** [filamentphp/filament#2243] [Help] [answered]
Hi all,

I do have a resource which is based on a model that is "view only", no frontend user can edit those values, they ship via seeder.
Hence, the only policy classes not returning false are view_any and view. Though, this does not lead to "clickable" entries in the resource table.

Am I blind not finding the option to simply "view" the resource entries or is the intended way to create a custom view and an action?

Thank you!

**Comment by vv-ac** [comment, marked as answer]
Got it, found it - here you go: https://filamentphp.com/docs/2.x/admin/resources#view-page

**Reply by khanakia** [reply]
@riskmate-ac It creates the view page but there is no data on the page. How do I add the data ?


<img width="908" alt="Screenshot 2022-07-20 at 5 45 02 PM" src="https://user-images.githubusercontent.com/2174170/179979363-e76c2236-7726-4e5b-9957-94b0302d4eb5.png">

**Reply by vv-ac** [reply]
Hi @khanakia,

I did not look it up and could not touch programming for a longer time, but I guess you need to define the form fields --> those will then be displayed as view only.
Other than that Filament (always?) provides you with the option to use a custom view.
